### DISCUSSION TITLE
Feature: Inventory Tabs for Item Categories

### DISCUSSION THREAD

**Discussion by Arufonsu** [AscensionGameDev/Intersect-Engine#1484] [Ideas]
### **Is your feature request related to a problem? Please describe.**
It's related to an enchancement regardless the Organization of Items in the Inventory.
This is related to keeping things organized _**along** with the equipable ammo_ feature requested by [@ghost](https://github.com/ghost) ( #253 ) and Equipment Inventory Independency from Item Inventory that i requested in #524.

There's many MMORPGs that use inventory tabs, one specific tab for equipment, one for consumables, one for ammo, another one for misc items, quest items, etc, etc. Basically one tab for each type of item within the inventory. Many of these MMORPGs follow this structure **in order to avoid having everything mixed up in one single window** (which usually leads to confusion among the players), i have played dozens of games like this in the past and i think that the implementation of this will be useful to improve the item's organization.

### **Describe the solution you'd like**
**_It would be a nice solution to implement inventory tabs, where items can be sorted out following their type of item in order to enchance the organization of items inside the inventory. This engine already allows the devs to have different types of items on the go, we could probably take adventage of that fact and allow it to organize the items based what we already have!._**

_`Few examples of inventory tabs in popular MMORPGs.`_

![skin-and-gemstone](https://user-images.githubusercontent.com/17498701/107855283-2a988680-6dd6-11eb-8463-ccb9b14086c4.png)

![Ragnarok_Online_Inventory_Window](https://user-images.githubusercontent.com/17498701/107862182-e91cd100-6dff-11eb-986f-5ee91aee79f8.gif)

![2](https://user-images.githubusercontent.com/17498701/107855284-2bc9b380-6dd6-11eb-8644-112ad97ea2cc.png)
![rh7Kqgo](https://user-images.githubusercontent.com/17498701/107855285-2c624a00-6dd6-11eb-8134-70bf2eee0a0c.png)

### **Additional context**
Having these organization-friendly features as **optional** for those devs who wish to have their player's inventory sorted out differently would be nice, you could always opt out of these features anyway.

**Comment by da-rulez** [comment]
Seems pretty game specific. Some games are built around inventory management.

-1
